lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -2689,15 +2686,35 @@ def test_user_service_with_anonymous_user(self):
         assert not runtime.user_is_staff
 
     def test_render_template(self):
-        runtime, _ = render.get_module_system_for_user(
-            self.user,
-            self.student_data,
-            self.descriptor,
-            self.course.id,
-            self.track_function,
-            self.xqueue_callback_url_prefix,
-            self.request_token,
-            course=self.course,
-        )
-        rendered = runtime.render_template('templates/edxmako.html', {'element_id': 'hi'})  # pylint: disable=not-callable
+        rendered = self.runtime.render_template('templates/edxmako.html', {'element_id': 'hi'})  # pylint: disable=not-callable
         assert rendered == '<div id="hi" ns="main">Testing the MakoService</div>\n'
+
+    @override_settings(COURSES_WITH_UNSAFE_CODE=[COURSE_ID])
+    def test_can_execute_unsafe_code(self):
+        assert self.runtime.can_execute_unsafe_code()
```

Comment:
Since this impacts security, can we add a test here as well for a course which is not in the list?

Reply:
Resolved with https://github.com/edx/edx-platform/pull/29260/commits/1c78884946ff950072315add3247fc546eca04eb#diff-6c688b209c164c25065d8a26e35b476b073fd6b4fba287c4317d084790a60a81R2696-R2699